cloudbuild/gae.yaml

```diff
@@ -1,11 +1,12 @@
 steps:
-- name: 'gcr.io/cloud-builders/gcloud'
-  args: ['-i -e "s/${_GCP_PROJECT}/$GCP_PROJECT/g"',
-  'app.yaml']
-  entrypoint: sed
+- name: 'ubuntu'
+  args: ['bash', './create_env.sh']
+  env:
+  - 'PROJECT=$PROJECT_ID'
+
 - name: 'gcr.io/cloud-builders/gcloud'
   args: ['app', 'deploy',
    '--no-promote',
    '--stop-previous-version',
-   '--project', '${_GCP_PROJECT}',
+   '--project', '${PROJECT_ID}',
    '--quiet']
```

Comment:
Hi @sang-d , the $PROJECT_ID is a built-in CloudBuild,
we would like to have a custom variables
`${_GCP_PROJECT}` is a custom variable,
we may have others variables like `SLACK_CLIENT_SEC , SLACK_CLIENT_ID`
according to the substitution here : https://cloud.google.com/cloud-build/docs/configuring-builds/substitute-variable-values , those variables should be ${_SLACK_CLIENT_SEC}, ${_SLACK_CLIENT_ID}

btw, could you please create move the PR to https://github.com/wizeline/kudos-me/issues/1
thanks

Reply:
sure, I see you are in case running build and deploying to different project